src/libraries/System.Diagnostics.Process/tests/ProcessTests.Unix.cs

```diff
@@ -833,6 +833,15 @@ public async Task Kill_ExitedNonChildProcess_DoesNotThrow(bool killTree)
                         // process still exists, wait some time.
                         await Task.Delay(100);
                     }
+
+                    DateTime now = DateTime.UtcNow;
+                    if (start.Millisecond + Helpers.PassingTestTimeoutMilliseconds <= now.Millisecond)
+                    {
+                        Console.WriteLine("{0} Failed to kill process {1} started at {2}", now, nonChildProcess.Id, start);
+                        Helpers.DumpAllProcesses();
+                    }
+
+                    Assert.True(start.Millisecond + Helpers.PassingTestTimeoutMilliseconds > now.Millisecond);
```

Comment:
Again need to subtract DateTime from DateTime to get a TimeSpan and then get TotalMilliseconds (not Milliseconds) off it.